docs/root/about_docs.rst

```diff
@@ -1,8 +0,0 @@
-About the documentation
```

Comment:
I see how this section could become useful as the docs get bigger. I feel fine about leaving it in, or resurrecting when needed. Up to you.

Reply:
I'd prefer to remove it for now and revive it later if needed (doesn't provide much value as-is IMO)